ui/src/clientLibraries/graphics/JSLogo.tsx

```diff
@@ -17,12 +17,12 @@ const JSLogo: SFC = () => {
            .js2{fill-rule:evenodd;clip-rule:evenodd;fill:#FFFFFF;}
            .js3{fill-rule:evenodd;clip-rule:evenodd;fill:#699F63;}
            .js4{clip-path:url(#XMLID_5_);}
-           .js5{fill:url(#SVGID_1_);}
-           .js6{fill:url(#SVGID_2_);}
-           .js7{fill:url(#SVGID_3_);}
-           .js8{fill:url(#SVGID_4_);}
-           .js9{fill:url(#SVGID_5_);}
-           .js10{fill:url(#SVGID_6_);}
+           .js5{fill:#699F63;}
+           .js6{fill:#699F63;}
+           .js7{fill:#699F63;}
+           .js8{fill:#699F63;}
+           .js9{fill:#699F63;}
+           .js10{fill:#699F63;}
         `}
       </style>
       <rect className="js0" width={300} height={300} />
```

Comment:
Can we rename these classes to be a bit more descriptive?

Reply:
this matches the format of our other logo files.